models/org.go

```
@@ -485,10 +484,14 @@ func AddOrgUser(orgID, uid int64) error {
 	}
 
 	if _, err := sess.Insert(ou); err != nil {
-		sess.Rollback()
+		if err = sess.Rollback(); err != nil {
```

The first error is lost.